Ginger/GingerCoreCommon/GeneralLib/ObservableListAttributes.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Amdocs.Ginger.Common.GeneralLib
+{
+    class ObservableListAttributes
+    {
+    }
+
+    public class IsLazyLoadAttribute : Attribute
+    {
+        public IsLazyLoadAttribute()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5012537)